packages/jest-repl/src/cli/repl.ts

```diff
@@ -69,17 +72,15 @@ if (jestProjectConfig.transform) {
   }
 }
 
-const replInstance = repl.start({
+const replInstance: repl.REPLServer = repl.start({
   eval: evalCommand,
   prompt: '\u203A ',
   useGlobal: true,
-});
+} as repl.ReplOptions);
```

Comment:
Compiler complains if I don't cast this.

Reply:
are we missing any options?

Reply:
or are the typings wrong?

Reply:
I think my typing of evalCommand is off. Lemme try and see if I can type it ebetter
![image](https://user-images.githubusercontent.com/18214059/53493513-757c3f00-3ad6-11e9-9f1f-2e1a7d2c82bb.png)